### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by viru2001**
# Added Product listing page

This page consists of 2 sections :
1. filters section
2. products list section

e com product listing page

[Deploy Preview](https://deploy-preview-5--gamers-bazaar.netlify.app/pages/products.html)

### CODE REVIEW

**Review by abhijeetekad — APPROVED** [review]
The code looks good to merge.

**Review by nisargpawade — APPROVED** [review]
the cards look a bit long, else looks good. okay to merge

File: pages/products.html [hunk 1]
```diff
@@ -0,0 +1,877 @@
...
+          <span class="vertical-card-badge text-sm p-3 p-relative"
+            >Best Seller</span
+          >
+          <div class="p-4">
+            <img
+              class="img-responsive d-block m-auto"
+              src="../assets/readgear-headphones.png"
+              alt="headphone"
+            />
+          </div>
```
Comment (nisargpawade) [inline pages/products.html]: see if you can cut some height here

**Review by nisargpawade — APPROVED** [review]

File: pages/products.html (hunk 1 above)
Comment (viru2001) [reply pages/products.html]: okay. will reduce size of the image and remove extra spacing. thank you for giving suggestion

**Pull request merged**